ui/src/components/credentials/LinkDeleteModal.tsx

```diff
@@ -0,0 +1,50 @@
+import { Modal, Typography, message } from "antd";
+
+import { deleteLink } from "src/adapters/api/credentials";
+import { ReactComponent as IconClose } from "src/assets/icons/x.svg";
+import { useEnvContext } from "src/contexts/env";
+
+export function LinkDeleteModal({
+  id,
+  onClose,
+  onDelete,
+}: {
+  id: string;
+  onClose: () => void;
+  onDelete: () => void;
+}) {
+  const env = useEnvContext();
+
+  const handleDeleteLink = () => {
+    void deleteLink({ env, id }).then((response) => {
+      if (response.isSuccessful) {
+        onClose();
+        onDelete();
+
+        void message.success(response.data);
+      } else {
+        void message.error(response.error.message);
+      }
+    });
+  };
+
+  return (
+    <Modal
+      cancelText="Close"
+      centered
+      closable
+      closeIcon={<IconClose />}
+      maskClosable
+      okButtonProps={{ danger: true }}
+      okText="Delete"
+      onCancel={onClose}
+      onOk={handleDeleteLink}
+      open
+      title="Are you sure you want to delete this credential link?"
+    >
+      <Typography.Text type="secondary">
+        Users will not be able to receive this credential anymore. This action cannot be undone.
```

Comment:
Small typo: "anymore" should be "any more" (designs are wrong too), in fact it should actually say "any longer", since that is grammatically correct.

Reply:
are you sure?

<img width="655" alt="image" src="https://user-images.githubusercontent.com/15896177/232749135-ef8a34fe-6985-42a4-882a-3be27138508b.png">

Reply:
Indeed, you are right. "Anymore" refers to time, and "any more" to quantities according to [Grammarly](https://www.grammarly.com/blog/anymore-vs-any-more/).

Nevertheless, it seems that "any longer" is [more correct here](https://learningenglish.voanews.com/a/anymore-any-longer-and-no-longer/6315945.html).